app/assets/stylesheets/header.scss

```diff
@@ -1,429 +1,341 @@
+$mrgn_cntr_height: $head_height - 3 * $mrgn_cntr_item;
+$img_height: 0.80*$head_height;
 
 body > header {
-  box-shadow: 0 1px 3px rgba(0,0,0,0.9);
   background: image-url('header-bg.png') rgb(40,35,35);
   z-index: 1001;
-  padding: 6px 0;
-  color: #CCC;
-  height: 26px;
+  height: $head_height;
   position: fixed;
   width: 100%;
-  min-width: 620px;
   top: 0;
   left: 0;
-  border-bottom: 1px solid #000;
-
-  > div > div.container {
-    height: 26px;
-  }
 
   .diaspora_header_logo {
     float: left;
-    margin-top: -6px;
-    height: 38px;
+    height: $head_height;
     width: 65px;
   }
 
-  .ie-user-menu-active { height: 150px; }
+  nav.navbar.navbar-default{ margin: 0; }
 
-  a.header_root_link {
-    display: inline-block;
-    margin-top: 5px;
-  }
+  > header, nav, div.container-fluid, ul#user-nav, li#conversations-badge{ height: $head_height; }
 
-  a {
-    color: #CCC;
-    color: rgb(147,147,147);
+  #user-menu, #user-nav > li{
+    &:hover{ background-color: $hover_bg_color; }
+  }
 
-    &:hover, &:focus {
-      background: none;
-      color: $highlight-white;
-    }
-    &:focus {
-      outline: thin dotted $border-dark-grey;
-      text-decoration: none;
+  #notifications-dropdown, #user-menu > ul {
+    a:hover, a:focus {
+      background: linear-gradient(to bottom, $blue, $dark-blue);
     }
   }
 
-  &.landing {
-    height: 40px;
-    .right { top: 10px; }
-  }
+  /* General rules for header dropdowns */
+  /***************************************
+   * ".header-dropdown" to be changed to *
+   * ".header-dropdown-right" with BS3   *
+   ***************************************/
+  div.header-dropdown{
+    background: transparent;
+    height: $head_height;
+    border: none;
+    float: right;
+    padding: 0 6px;
+    margin: 0;
 
-  #nav_badges {
-    display: inline-block;
-    margin-top: 2px;
-    float: left;
-    width: 61px;
+    &.open{
+      background-color: $hover_bg_color;
+    }
 
-    a:hover { text-decoration: none; }
+    img{
+      height: $img_height;
+      margin: 0 1em 0 0;
+      padding: 0;
+    }
 
-    .badge {
-      width: 22px;
-      position: relative;
-      top: 2px;
-      display: inline;
-      margin: 0 2px;
-      font-weight: bold;
-      font-size: smaller;
+    button{
+      background: transparent;
+      color: $font_color;
+      border: none;
+      height: 100%;
+      margin: 0;
+      padding: 0;
+
+      #caret{
+        margin: 0;
+        padding: 0;
+        color: $font_color;
+      }
 
-      .badge_count {
+      #user-id{
+        margin-right: 1em;
+      }
+
+      div.badge_count {
+        background-color: $red;
         border-radius: 2px;
-        z-index: 3;
         position: absolute;
-        top: -4px;
-        left: 15px;
+        line-height: 14px;
         padding: 1px 2px;
-        background-color: $red;
-        line-height: 12px;
-        color: #fff;
+        font-size: 12px;
+        height: 1em;
+        color: #FFF;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`